src/organisms/tables/ComparisonTable/comparison_table.module.scss

```diff
@@ -0,0 +1,63 @@
+@import 'src/assets/stylesheets/global';
+
+.comparisonTable {
```

Comment:
@drewdrewthis that's my bad, I should have checked what the standard was - changed to kebab case